### PR TITLE
BGA-62 forwarder deploy

### PR DESCRIPTION
**Pull request by gonzaalvarez70**
Added forwarder deploy as address initialization transaction.

### CODE REVIEW

File: src/coin/eth/transactionBuilder.ts [hunk 1]
```diff
@@ -346,6 +352,53 @@ export class TransactionBuilder extends BaseTransactionBuilder {
...
+    this._contractCounter = contractCounter;
+  }
+
+  /**
+   * Build a transaction to create a forwarder.
+   *
+   * @returns {TxData} The Ethereum transaction data
+   */
+  private buildAddressInitializationTransaction(): TxData {
+    const addresInitData = getAddressInitializationData();
```
Comment (marktoda) [inline src/coin/eth/transactionBuilder.ts:376]: nit: spelling

File: src/coin/eth/transactionBuilder.ts [hunk 2]
```diff
@@ -346,6 +352,53 @@ export class TransactionBuilder extends BaseTransactionBuilder {
...
+  }
+
+  /**
+   * Obtain the inferred forwarder address for an Address initialization transaction
+   * determined by the contract address and the contract counter.
+   *
+   * @returns {string} the forwarder contract address
+   */
+  public getForwarderAddress(): string {
+    if (this._type != TransactionType.AddressInitialization) {
```
Comment (marktoda) [inline src/coin/eth/transactionBuilder.ts:389]: `!==` to avoid type coersion

File: src/coin/eth/utils.ts [hunk 3]
```diff
@@ -89,6 +89,16 @@ export function sendMultiSigData(
   return addHexPrefix(Buffer.concat([method, args]).toString('hex'));
 }
 
+/**
+ * Returns the create forwarder method calling data
+ *
+ * @returns {string} - the createForwarder method encoded
+ */
+export function getAddressInitializationData(): string {
+  const method = EthereumAbi.methodID('createForwarder', []);
```
Comment (marktoda) [inline src/coin/eth/utils.ts:98]: nit: We have this statically saved in `walletUtils` I think, so you don't have to re-calculate it over and over. Not a big deal though..

File: src/coin/eth/transactionBuilder.ts (hunk 1 above)
Comment (gonzaalvarez70) [reply src/coin/eth/transactionBuilder.ts:376]: Fixed, nice catch!

File: src/coin/eth/transactionBuilder.ts (hunk 2 above)
Comment (gonzaalvarez70) [reply src/coin/eth/transactionBuilder.ts:389]: Great, done.

File: src/coin/eth/utils.ts (hunk 3 above)
Comment (gonzaalvarez70) [reply src/coin/eth/utils.ts:98]: Sure, I added it just so it's explicit. But I'll use the static constant. I'll just keep the method `getAddressInitializationData()` to keep the structure consistent.

**Pull request merged**